eng/common/LiveTestResources/New-TestResources.cmd

```diff
@@ -0,0 +1,17 @@
+@echo off
```

Comment:
I think we should use the naming convention for PowerShell modules, i.e. the parent directory is the same as the base name for scripts' nouns, i.e. "TestResources".